lms/djangoapps/instructor/tests/test_legacy_raw_download_csv.py

```diff
@@ -104,6 +107,147 @@ def test_grade_summary_data(self):
             ]
         }
 
+        non_grade_columns = 5
+        if (get_grades is False) or (use_offline is True):
```

Comment:
Nit: You don't need these parens in Python.

Error: Use `==` not `is`. `is` is identity testing, `==` is equality testing.